internal/common/rules/globalrulescache.go

```diff
@@ -7,6 +7,8 @@ import (
 
 var AvailableRules = map[string]RuleMatrix{}
 
+var RulesInPlay = map[string]RuleMatrix{}
```

Comment:
![image](https://user-images.githubusercontent.com/33488131/102371191-d596ea80-3fb5-11eb-97d3-cc0aef504908.png)

Does GoLand not have these warnings for the standard go style? these are really nice because they show up nicely on hover:

![image](https://user-images.githubusercontent.com/33488131/102371269-ea737e00-3fb5-11eb-95b2-1c06e5bf2a7c.png)

Reply:
So apparently `golint` is frozen and deprecated a year a go and people have been moving to `golangci-lint` (which was what we have in the automated CI lint here). GoLand also does not format `golint`. I think the `should have comment` thing can be annoying so follow at your own volition, I suppose; it's a nice to have so that we don't need to go to the file to figure out what the function/variable does. Self-documenting code wins as well. 

Feel free to ignore the original comment :)

Reply:
Ok sure, I'll comment the variables anyway, but like ou said I can see that not being the most popular standard to stick to